tests/migrator/test_books.py

```diff
@@ -1077,27 +1077,26 @@ def test_related_record(app):
                 },
             },
         )
-        with pytest.raises(ManualImportRequired):
-            check_transformation(
-                """
-                <datafield tag="787" ind1=" " ind2=" ">
-                    <subfield code="i">Random text</subfield>
-                    <subfield code="w">7483924</subfield>
-                </datafield>
-                """,
-                {
-                    "_migration": {
-                        **get_helper_dict(),
-                        "has_related": True,
-                        "related": [
-                            {
-                                "related_recid": "7483924",
-                                "relation_type": "other",
-                            }
-                        ],
-                    },
+        check_transformation(
```

Comment:
unfortunately this is incorrect, we are not testing books model but journals, it should stay as it was